tests/python/relay/test_pass_manager.py

```diff
@@ -559,15 +569,20 @@ def test_print_debug_callback():
         ]
     )
 
-    assert __TRACE_COUNTER__ == 0
     mod = tvm.IRModule({"main": func})
 
-    with tvm.transform.PassContext(opt_level=3, trace=_tracer):
+    pass_counter = PassCounter()
+    with tvm.transform.PassContext(opt_level=3, instruments=[pass_counter]):
+        # Should be reseted when entering pass context
+        assert pass_counter.get_counts() == 0
         mod = seq(mod)
 
-    # TODO(@jroesch): when we remove new fn pass behavior we need to remove
-    # change this back to 3
-    assert __TRACE_COUNTER__ == 5
+        # TODO(@jroesch): when we remove new fn pass behavior we need to remove
+        # change this back to match correct behavior
+        assert pass_counter.get_counts() == 6
```

Comment:
how come this number changed?

Reply:
As mentioned before, FunctionPassNode's InfterType() and SequentialPass() are not traced before.
In this PR, we override pass::operator() to instrument instead of instrumenting in subclasses::operator().

In this test case, sequential pass now is instrumented (counted).